lib/membrane_caps_audio_raw.ex

```diff
@@ -283,6 +283,27 @@ defmodule Membrane.Caps.Audio.Raw do
     end
   end
 
+  @doc """
+  Returns a binary which corresponds to the silence during the given interval
+  of time in given caps' fromat
+
+  ## Examples:
+  The following code generates one second of the silence for the given caps
```

Comment:
It actually does generate 100 microseconds ;) I bet you changed this below and forgot to change here, and that's why such comments should be avoided